### DISCUSSION TITLE
Can a checkbox option label contain HTML?

### DISCUSSION THREAD

**Discussion by mprofitlich** [verbb/formie#1762] [General]
Hello!
Is ist possibe to add HTML code to a option label? We would like to link the terms and conditions form there. We tried to write HTML in the backend and we tried the following overwrite. Both failed.

```
{% set form = craft.formie.forms.handle('kontaktdaten').one() %}
{% do form.setFieldSettings('agb', {
	options: [
		{ label: 'Override <a href="/alaa">One</a>' }
	],
}) %}
{{ craft.formie.renderForm(form) }}
```

Thanks for your help!
Moritz

**Comment by engram-design** [comment]
No you can't - you wouldn't be able to click on the HTML in an `<option>` element, and I'm pretty sure that's invalid HTML.

Your best bet is to add it to the Instructions for the field.

**Reply by mprofitlich** [reply]
Thanks, @engram-design. I believe I asked wrongly and I should have added a screenshot.
<img width="287" alt="grafik" src="https://github.com/verbb/formie/assets/6146601/d60611bb-f220-48e6-b26b-fac8df85129b">
I discovered what I was looking for: The field type ‹Agree› allows for writing HTML in the label.

For future use: Can I swap the content of any other label to HTML?

**Reply by engram-design** [reply]
Ah, gotcha. Yes for a Checkboxes field, that's pretty valid, I was thinking purely for a Dropdown field.

There aren't any plans to support full HTML to labels to options fields, purely due to UI constraints and over-complicating things. We might just add Markdown support to keep things simple.
